src/btree/bt_sync.c

```diff
@@ -89,6 +92,9 @@ __sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 
 	internal_bytes = leaf_bytes = 0;
 	internal_pages = leaf_pages = 0;
+	int_bytes_svpt = leaf_bytes_svpt = 0;
```

Comment:
convention is to initialize variables closest to the first use, maybe these can move just above the for loop where these variables get used.